h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -791,6 +792,18 @@ public RegressionModelPrediction predictRegression(RowData data, double offset)
     return p;
   }
 
+  public KLimeModelPrediction predictKLime(RowData data) throws PredictException {
+    double[] preds = preamble(ModelCategory.KLime, data);
```

Comment:
I think this expect to that KLime overrides `getModelCategories` - here we pass `ModelCategory.KLime` which is checked against `KLime#getModelCategories` which returns `[Regression]`.

CC: @navdeep-G